Modules/xxmodule.c

```diff
@@ -358,31 +358,32 @@ xx_exec(PyObject *m)
 
     /* Finalize the type object including setting type of the new type
      * object; doing it here is required for portability, too. */
-    if (PyType_Ready(&Xxo_Type) < 0)
-        goto fail;
+    if (PyType_Ready(&Xxo_Type) < 0) {
+        return -1;
+    }
 
     /* Add some symbolic constants to the module */
     if (ErrorObject == NULL) {
```

Comment:
ErrorObject is not used outside this function. Would it make sense to simply remove this global variable?

Reply:
Maybe. It depends on what we want the xxmodule.c to communicate and what we would like to show off. It's out of scope for this PR anyway. We could elevate such a discussion to Discourse, I guess.